src/firmware_new/boards/arduino_uno/echo_library.h

```diff
@@ -0,0 +1,9 @@
+
+#ifndef SRC_ECHO_LIBRARY_H
+#define SRC_ECHO_LIBRARY_H
```

Comment:
nit: can we use `#pragma once` here (its okay if we can't)

Reply:
we can, changes made